FileOrganization.md

```diff
@@ -1,5 +1,41 @@
 # File Organization
 
+## MARK Comments
+
+The special `// MARK: Some text` comment should be used to group related methods into sections (`#pragma mark` in objc).
+
+
+<img width="529" alt="Example of using the MARK syntax in comments to split the navigator into sections" src="https://user-images.githubusercontent.com/1363082/154284314-33610b82-62c0-4840-a037-7eba8b8a3269.png">
```

Comment:
Does this screenshot only contain functions from a publicly viewable repo?

Reply:
It's technically internal, but since the symbols can be pulled from the binary anyway, there are no concerns here. (Also, this screenshot is so old, the file it shows doesn't even exist any more)